src/workerd/api/crypto-impl-aes-test.c++

```diff
@@ -96,16 +94,14 @@ KJ_TEST("AES-CTR key wrap") {
   // wrap if it didn't have "encrypt" in its usages when created.
 
   jsg::test::Evaluator<CryptoContext, CryptoIsolate> e(v8System);
-  CryptoIsolate &cryptoIsolate = e.getIsolate();
-  jsg::runInV8Stack([&](jsg::V8StackScope& stackScope) {
-    CryptoIsolate::Lock isolateLock(cryptoIsolate, stackScope);
+  e.getIsolate().runInLockScope([&](CryptoIsolate::Lock& isolateLock) {
     auto isolate = isolateLock.v8Isolate;
 
     isolateLock.withinHandleScope([&] {
       auto context = isolateLock.newContext<CryptoContext>().getHandle(isolateLock);
       auto contextScope = isolateLock.enterContextScope(context);
 
-      auto& js = jsg::Lock::from(isolate);
+      jsg::Lock& js = isolateLock;
```

Comment:
Nit: With this being done, can the remaining instances of `jsg::Lock::from(isolate)` also be cleaned up or is `Lock::from` needed there?

Reply:
Much of this is getting cleaned up in follow up PRs. Stay tuned.